src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <section class="stopwatch stopwatch--speed-up">
+      <div class="stopwatch__minutes"></div>
+      <div class="stopwatch__seconds"></div>
+      <div class="stopwatch__center"></div>
```

Comment:
Check if all the BEM naming conventions are followed correctly as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Ensure that each element is named according to its block and its role.